lib/flagship/context.rb

```diff
@@ -25,14 +21,24 @@ def method_missing(name, args = [], &block)
   end
 
   def respond_to_missing?(name, include_private = false)
-    @values.key?(name) or super
+    values.key?(name) or super
```

Comment:
`current_values`?

Now I noticed that this method is not covered by any tests 😇